src/admin/forms/BannerForm.js

```diff
@@ -0,0 +1,217 @@
+import React, {useState} from 'react';
+import {withRouter} from 'react-router-dom';
+import {Button, Form, Message} from 'semantic-ui-react';
+
+const SEVERITY = [
+  {key: 'INFO', name: 'Info'},
+  {key: 'WARN', name: 'Warning'},
+  {key: 'ERROR', name: 'Error'},
+];
+
+const BannerForm = ({history, isUpdate, mutation, banner, loading, err}) => {
+  const [submitting, setSubmitting] = useState();
+  const [error, setError] = useState(err);
+  const [state, setState] = useState({
+    message: banner.message,
+    enabled: banner.enabled,
+    startDate: banner.startDate ? banner.startDate.split('T')[0] : '',
```

Comment:
I think you should be able to interpret these ISO8601 dates with `new Date(banner.startDate)` and use `toLocaleString` for formatting.